### ISSUE TITLE
Importing makeMockModels yields error that .sequelizerc can not be found

### ISSUE THREAD

**Issue by dschuessler** [davesag/sequelize-test-helpers#91]
I am importing `makeMockModels` as follows:

```Typescript
import { makeMockModels } from 'sequelize-test-helpers';
```

Running my Mocha test yields the following error originating from `src/mockModels.js`, line 11: https://gist.github.com/dschuessler/0e1c075837c4d7207151753313950d42

When I copy the exact path from the error and output it with `cat /api/backend/.sequelizerc` the file is present.

The error disappears when I change line 11 from:

```Javascript
 sequelizerc = require(path.join(projectRoot, '.sequelizerc'))
```

to

```Javascript
  sequelizerc = require.main.require('./.sequelizerc')
```

I am not entirely sure about the nature of the problem, but I guess it has something to do with Node having problems to require from absolute paths. I would greatly appreciate it, if you could have a closer look. Thanks in advance!

**Comment by davesag** [comment]
Hi @dschuessler — a few questions to help me debug this: I've never known Node to have issues with a require from an absolute path, but it might be something to do with whatever pre-processor you are using to convert typescript to javascript.

1. What version of Node are you using? (`node -v`)
2. what is your project root?  (I am guessing it's `/api/backend` going on your logs but can you please confirm this)
3. what is in `/api/backend/.sequelizerc`?
4. Does sequelize itself find your `.sequelizerc` file?

I have made some minor updates to how it handles the error if it can't find the `.sequelizerc` file so please try your code against the latest `develop` branch instead of the version in `npm` and let me know how that goes.

**Comment by dschuessler** [comment]
Your commit has the same effect on my test as my workaround. The test now runs through and fails with the following error.

```
     Error: ENOENT: no such file or directory, scandir '/api/backend/src/models'
      at Object.readdirSync (fs.js:785:3)
      at listModels (node_modules/sequelize-test-helpers/src/mockModels.js:32:6)
      at finder (node_modules/sequelize-test-helpers/src/mockModels.js:36:26)
      at Object.makeMockModels (node_modules/sequelize-test-helpers/src/mockModels.js:39:6)
      at Object.<anonymous> (src/controller/user/user.service.spec.ts:26:26)
      at Generator.next (<anonymous>)
      at /api/backend/src/controller/user/user.service.spec.ts:7:71
      at new Promise (<anonymous>)
      at __awaiter (src/controller/user/user.service.spec.ts:3:12)
      at Context.it.only (src/controller/user/user.service.spec.ts:24:72)
```

The content of `.sequelizerc` is the following:

```Javascript
const path = require('path');

module.exports = {
  'config': path.resolve('config', 'sequelize-cli.js')
  'models-path': path.resolve('models')
}
```

So `makeMockModels` does not read the model path from `.sequelizerc`, but searches in the standard location.

Regarding your other questions:

1. Node 10.15.3
2. Yes, the root directory is `/api/backend`
4. I am pretty new to this tech stack so I actually don't know how to find out if Sequelize can find `.sequelizerc`. If you tell me how I can find out, I will gladly have a look. I can assure that Sequelize does find its models.

Also Typescript is transpiled by means of `ts-node` 8.1.0.

**Comment by davesag** [comment]
@dschuessler Okay so your models are in `/api/backend/models/` and not in `/api/backend/src/models`. I presume that's where the `.ts` files are though and not where the resulting `.js` files end up, or do you compile your `.js` into the same path as the `.ts` files?

My guess is that this is come sort of typescript weirdness as this library works just fine if you just use javascript.  Are you able to share your repo, or make a small demo repo that exhibits this bug that I could take a look at? I'm not a typescript person but I don't see any reason why basic things like `require` would fail.  Also what platform are you running on?  Linux, Mac, or Windows. That _might_ be a factor.

**Comment by davesag** [comment]
@dschuessler I have pushed version 1.1.1 to npm — it probably doesn't fix your issue but it does include the improvement to how I detect the missing `.sequelizerc` file.

**Comment by dschuessler** [comment]
I quickly assembled a small demo repo reproducing the error:

https://github.com/dschuessler/sequelize-test-helpers-bug-example

Do `npm install && npm test` to get the error I get.

**Comment by davesag** [comment]
Thanks I'll aim to take a look over the weekend.

**Comment by davesag** [comment]
Hey @dschuessler please see https://github.com/dschuessler/sequelize-test-helpers-bug-example/pull/1 where I have fixed your tests and example code.

**Comment by davesag** [comment]
I've released version 1.1.2 which improves the documentation of custom models folder path and adds the ability to supply custom file suffixes as well
